src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,369 @@
+import * as React from 'react'
+import * as PropTypes from 'prop-types'
+
+import { Extendable } from '../../../types/utils'
+import { ComponentSlotStyle } from '../../themes/types'
+import Downshift, { DownshiftState, StateChangeOptions } from 'downshift'
+import Label from '../Label/Label'
+import { UIComponent, customPropTypes } from '../../lib'
+import Input from '../Input/Input'
+import keyboardKey from 'keyboard-key'
+import List from '../List/List'
+import Text from '../Text/Text'
+import Image from '../Image/Image'
+import ListItem from '../List/ListItem'
+import _ from 'lodash'
+import Icon from '../Icon/Icon'
+
+export interface DropdownProps {
+  className?: string
+  fluid?: boolean
+  items?: DropdownListItem[]
+  multiple?: boolean
+  onChange?: (active: DropdownListItem[]) => any
+  placeholder?: string
+  search?: boolean
+  styles?: ComponentSlotStyle<DropdownProps, DropdownState>
+  toggleButton?: boolean
+}
+
+export interface DropdownState {
+  active: DropdownListItem[]
```

Comment:
`active` and `inputValue` should be autocontrolled props.

Reply:
made the whole component autocontrolled and made necessary changes